community/graph-algo/src/main/java/org/neo4j/graphalgo/impl/centrality/EigenvectorCentralityArnoldi.java

```diff
@@ -95,6 +96,14 @@ public EigenvectorCentralityArnoldi( Direction relationDirection,
      */
     public Double getCentrality( Node node )
     {
+        if ( !nodeSet.contains( node ) )
+        {
+            return null;
+        }
+        if ( !node.hasRelationship() )
+        {
+            return 0d;
```

Comment:
This zero is not a flag for anything, it is the actual mathematical centrality value of a node without relationships. Still propose a constant?